apps/dsb-client-gateway-api/src/app/modules/message/service/message.service.ts

```diff
@@ -448,6 +448,9 @@ export class MessageService {
         clientGatewayMessageId
       );
     } else {
+
+      fs.promises.mkdir(this.uploadPath, { recursive: true }).catch(error => { this.logger.debug('caught exception : ', error.message); });
```

Comment:
use `fs.mkdirSync` for that, why you need recursive here?

This should not be created whenever message is posted/downloaded - only during application bootstrap

Reply:
Agree